src/modules/Workspaces/WorkspacesWindowArranger/WindowArranger.cpp

```diff
@@ -103,6 +203,67 @@ WindowArranger::WindowArranger(WorkspacesData::WorkspacesProject project) :
     m_ipcHelper(IPCHelperStrings::WindowArrangerPipeName, IPCHelperStrings::LauncherArrangerPipeName, std::bind(&WindowArranger::receiveIpcMessage, this, std::placeholders::_1)),
     m_launchingStatus(m_project)
 {
+    std::vector<HWND> movedWindows;
+    std::vector<WorkspacesData::WorkspacesProject::Application> movedApps;
```

Comment:
They are used only inside the `if (project.moveExistingWindows)` block, so please move them into the appropriate scope.